File: extract.py
import sys
try:
    import fitz # PyMuPDF
    doc = fitz.open('teste gil.pdf')
    text = ""
    for page in doc:
        text += page.get_text("text") + "\n\n"
    with open('pdf_text_pymupdf.txt', 'w', encoding='utf-8') as f:
        f.write(text)
    print("Extraction complete")
except Exception as e:
    print(f"Error: {e}")

File: extract.js
const fs = require('fs');
const pdf = require('pdf-parse');

let dataBuffer = fs.readFileSync('teste gil.pdf');

const parser = pdf;

parser(dataBuffer).then(function (data) {
    fs.writeFileSync('pdf_text.txt', data.text);
    console.log("Wrote to pdf_text.txt");
}).catch(e => console.error(e));
